core/src/components/action-sheet/action-sheet.tsx

```diff
@@ -128,22 +128,25 @@ export class ActionSheet implements ComponentInterface, OverlayInterface {
 
   /**
    * Dismiss the action sheet overlay after it has been presented.
+   *
+   * @param data Any data to emit in the dismiss events.
+   * @param role The role of the element that is dismissing the action sheet. For example, 'cancel' or 'backdrop'.
```

Comment:
Is there a set of accepted roles we can reference?

Reply:
It depends on the component, but all of the interfaces accept any string as well. Action sheet has the following:

```
role?: 'cancel' | 'destructive' | 'selected' | string;
```

Where alert is just `string`